covid_xprize/validation/scenario_generator.ipynb

```diff
@@ -128,7 +140,7 @@
    "outputs": [],
    "source": [
     "# Write to file\n",
-    "# scenario_df.to_csv(output_file, index=False)"
+    "scenario_df.to_csv(output_file, index=False)"
```

Comment:
Now that the file is written, I'm not sure we want to override it each time we run this notebook.